### ISSUE TITLE
Optic cupping

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9787]
**Preferred term label:**
Optic cupping

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Seen in glaucoma.  This is the opposite of papilledema, with higher pressure inside the eye.

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
@sukram

is this a synonym for [Increased cup-to-disc ratio HP:0012796](https://hpo.jax.org/app/browse/term/HP:0012796) ?

**Comment by MickeySegal** [comment]
https://glaucoma.org/optic-nerve-cupping/  "Both people with and without optic nerve damage have optic nerve cupping, although those with glaucoma tend to have a greater cup-to-disc ratio. A cup to disc ratio greater than six-tenths is generally considered to be suspicious for glaucoma."

So it seems to be the same.

**Comment by pnrobinson** [comment]
adding as synonym.
